src/pybind/rados.py

```diff
@@ -831,6 +831,32 @@ def blacklist_add(self, client_address, expire_seconds=0):
         if ret < 0:
             raise make_ex(ret, "error blacklisting client '%s'" % client_address)
 
+class OmapIterator(object):
+    """Omap iterator"""
+    def __init__(self, ioctx, ctx):
+        self.ioctx = ioctx
+        self.ctx = ctx
+
+    def __iter__(self):
+        return self
+
+    def next(self):
+        """
+        Get the next key-value pair in the object
+        :returns: next rados.OmapItem
+        """
+        self.key = c_char_p()
+        self.value = c_char_p()
+        self.lens = c_int()
+        run_in_thread(self.ioctx.librados.rados_omap_get_next,
+                      (self.ctx, pointer(self.key), pointer(self.value),
+                       pointer(self.lens)))
+        if isinstance(self.value.value, str):
+            self.value.value = self.value.value[0:self.lens.value]
```

Comment:
probably should raise `StopIteration` in such a case `if self.value.value is None`?